docs/deployment/deployment-checklist.md

```diff
@@ -0,0 +1,42 @@
+---
+slug: checklist
+append_help_link: true
+title: Pre-deployment checklist
+description: tk
+tags:
+  - Deployment
+---
+
+Before starting the deployment set-up, use this checklist to ensure that:
+
+- You have **access** to the resources needed to carry out the deployment.
+- You and your organization agree on the **scope** of the deployment.
+
+## Permissions and  access 
+
+Ensure that you and your deployment team have sufficient permissions to:
+
+- Add or make changes to CI jobs.
+- Create access tokens in your source code manager.
+- Commit files to all repositories you want Semgrep to scan.
+- (Optional) View SSO configurations.
+- (Optional) Set up channels in your chosen notification method (Slack, email, or webhooks).
+- (For self-hosted repositories) Edit your firewall or VPN configuration's allowlist.
+
+## Resources
+
+Determine the following:
+
+- What roles or departments will use Semgrep.
+- The number of repositories you will scan with Semgrep.
+
+## Processes
+
+Determine the following:
```

Comment:
Similar to my previous comment -- it might be helpful to have a bit more context here?

Reply:
I've redone the deployment checklist to make it an actual table checklist with better information, i hope it resolves this.